web-client/src/views/Messages/MessageTable.tsx

```diff
@@ -0,0 +1,394 @@
+import {
```

Comment:
Here, as in `MessagesIndividualInbox`, `MessagesSectionOutbox`, etc., we just pass in the columns and filters we want into `MessageList`.